test/test_unit_ssl_service.py

```diff
@@ -3,61 +3,60 @@
 
 from conjur.api.ssl_client import SSLClient
 
-GITHUB_FINGERPRINT = "54:C5:A2:27:06:92:CF:D1:1C:24:A3:92:96:41:49:06:21:B0:45:30"
+GITHUB_FINGERPRINT = "84:63:B3:A9:29:12:CC:FD:1D:31:47:05:98:9B:EC:13:99:37:D0:D7"
```

Comment:
As part of this failing fingerprint test we could fetch the certificate as part of the test (you could fetch the cert as part of the test `openssl s_client -connect github.com:443 < /dev/null 2>/dev/null | openssl x509 -fingerprint -noout -in /dev/stdin`) and compare that to whatever the CLI fetches

Reply:
You're assuming that openssl is present

Reply:
in the test container there should be if not, add the dependency or find another way to fetch cert (: